drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -36,8 +38,9 @@ config ETH_STM32_HAL_RX_THREAD_PRIO
 
 config ETH_STM32_HAL_USE_DTCM_FOR_DMA_BUFFER
 	bool "Use DTCM for DMA buffers"
-	default y
-	depends on SOC_SERIES_STM32F7X
+	default y if SOC_SERIES_STM32F7X
+	default n if SOC_SERIES_STM32H7X
+	depends on SOC_SERIES_STM32F7X || SOC_SERIES_STM32H7X
```

Comment:
If default n for H7, why not leaving this as is ("depends on F7")?

Reply:
You're right. It was just remnant from dev stage.